README.md

```diff
@@ -22,7 +22,7 @@ A live version of this theme can be found
 ## Features
 The main features of this theme include:
 * Page load speed optimizations (scores 98/100 on PageSpeed)
-* Syntax highlighting with `highlightjs`
+* Syntax highlighting with `chroma`
```

Comment:
If you can add in the feature list that no javascript is being used that would be great! Something like: `* No JavaScript` is probably fine